src/Client.php

```diff
@@ -63,68 +69,75 @@ public function getFolders() : array
         return array_keys($this->hordeClient->listMailboxes('*'));
     }
 
-    /**
+     /**
      * Finds the emails matching the query. If $query is null, then it will fetch the emails in the inbox.
-     *
+     * @param Query $query Description
+     * @param boolean $peek sets the peek option, "false" fetched emails' state will be set to seen "true" no change of state default is true
      * @return Email[]
      */
-    public function getEmails(Query $query = null) : array
-    {
```

Comment:
The `{` should be on a new line (this is the PSR-2 coding standard). This comment applies also below on other methods.

Reply:
Ok IDE auto format will sort it out.